spec/lib/political_edition_identifier_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+RSpec.describe PoliticalEditionIdentifier do
+  describe ".policical_organisation_ids" do
+    it "returns an array of content_ids" do
+      uuid_regex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/
+      expect(PoliticalEditionIdentifier.political_organisation_ids)
+        .to all match(uuid_regex)
+    end
+  end
+
+  describe "#political?" do
+    let(:document_type) do
+      primary_organisation_field = build(:tag_field,
+                                         type: "single_tag",
+                                         id: "primary_publishing_organisation")
+      organisations_field = build(:tag_field,
+                                  type: "multi_tag",
+                                  id: "organisations")
+      role_appointments_field = build(:tag_field,
+                                      type: "multi_tag",
+                                      id: "role_appointments")
+
+      build(:document_type, tags: [primary_organisation_field,
+                                   organisations_field,
+                                   role_appointments_field])
+    end
+
+    context "when a edition is associated with a role appointment" do
+      it "returns true" do
```

Comment:
I remember a code review I had from Leena, suggesting that if a `context` only has one `it` block inside it, then it should probably just be an `it` block. Something like:

```ruby
it "returns true when an edition is associated with a role appointment"
```

I quite like the insinuation of using the `context` syntax - that in a specific context, X is expected to happen - but find it more useful as a wrapping block for setting `let` instance variables. In this case, we only actually begin to capture the context when we're already inside the `it` block. I do think removing the `context` keyword would allow each test to feel more self-contained.

Reply:
Yeah that seems a good shout for this block of code since we don't make use of any lets